services/agents/mysql_test.go

```diff
@@ -158,10 +182,16 @@ func TestMySQLdExporterConfigTablestatsGroupDisabled(t *testing.T) {
 			"--web.listen-address=:{{ .listen_port }}",
 		},
```

Comment:
let's add to this test check that we have new args `mysql.ssl-cert-file` and `mysql.ssl-key-file`

Reply:
Added.